### PR TITLE
Htv3hc/feat/trigger other workflow

### PR DESCRIPTION
**Pull request by huavanthong**
Hi Thomas,

I would like to update code to trigger the build of RobotFramework AIO at this repos following behavior below:
* A new-pull-request, or updated-pull-request coming request to **develop** branch in this repos will trigger **trigger-build-aio** and **status-build-aio**. 
```yaml 
  pull_request:
    types:
      - closed
      - opened
      - synchronize
    branches: 
      - develop
```
* **trigger-build-aio** job will use [Webhook ](https://docs.github.com/en/webhooks/about-webhooks-for-repositories) feature to send a curl POST to **test-fullautomation/RobotFramework_AIO**, and trigger pipeline on it. And it also sends info related to merge request branch and the current repository to request it
```
curl -X POST https://api.github.com/repos/test-fullautomation/RobotFramework_AIO/dispatches \
          --header 'authorization: Bearer ${{ secrets.PAT_THONG }}' \
          -H 'Accept: application/vnd.github.everest-preview+json' \
          --data '{"event_type": "Triggered by other workflow", "client_payload": { "repository": "'"$GITHUB_REPOSITORY"'", "branch": "'"$PULL_REQUEST_BRANCH"'" }}'
```
* **status-build-aio** job will wait the status of the latest pipeline of RobotFramework_AIO until it completely following command below:
```bash
$ gh api repos/test-fullautomation/RobotFramework_AIO/actions/runs --jq '.workflow_runs | sort_by(.created_at) | last | .status'
completed

# Output:
# Case 1: in_progress -> pipeline is running
# Case 2: completed -> pipeline is completed
```
And get the result of pipeline AIO by command below:
```bash
gh api repos/test-fullautomation/RobotFramework_AIO/actions/runs --jq '.workflow_runs | sort_by(.created_at) | last | .conclusion'
success

# Output:
# Case 1: success-> pipeline AIO build is succeed
# Case 2: failure-> pipeline AIO build is failed
```
If the status is **SUCCESS**, we can see the result on this pipeline at Step: Report Status with log info below:
```bash
    echo "Pipeline RobotFramework AIO success."
```
If the status is **FAIL**, we will retrieve the corresponding logs, along with a link to the RobotFramework pipeline results.
```bash
    echo "Pipeline RobotFramework AIO is not succeed."
    echo "Details at link: $pipeline_link"
```
Could you help me review this pull-request? 

Thank you and best regards,
Thong Hua

### CODE REVIEW

**Comment by ngoan1608** [comment]
Hi Thong,

I think it should be better that this feature is implemented in a new workflow (separated `*.yml` file).
So that it is easy for us to update, maintain and avoid side-effect.

Thank you,
Ngoan

File: .github/workflows/release_to_pypi.yml [hunk 1]
```diff
@@ -45,4 +55,64 @@ jobs:
             allowUpdates: true
             omitNameDuringUpdate: true
             makeLatest: true
-            artifacts: dist/*
\ No newline at end of file
+            artifacts: dist/*
+
+  trigger-build-aio:
```
Comment (ngoan1608) [inline .github/workflows/release_to_pypi.yml]: The trigger job is also executed when tagging for releasing PyPI. 
It is not expected.

**Review by ngoan1608 — CHANGES_REQUESTED** [review]

**Comment by huavanthong** [comment]
Hi Ngoan,

Thank you for your comments. I have updated it. 
Please take it for your info.

Best regards,
Thong Hua

**Review by test-fullautomation — APPROVED** [review]
Hi Thong,
thank you for your pull-request.
I will merge it immediately once we released 0.9.0.
Thank you,
 Thomas

**Comment by test-fullautomation** [comment]
Hi @huavanthong ,
I will need this change for all repositories, correct?
Thank you,
 Thomas

**Pull request merged**